src/main/java/net/imagini/zkcli/CliParameters.java

```diff
@@ -3,74 +3,169 @@
 import com.beust.jcommander.JCommander;
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.internal.Lists;
+import net.imagini.jzookeeperedit.ZkClusterManager;
+import org.apache.curator.framework.CuratorFramework;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 public class CliParameters {
     @Parameter
-    List<String> positionalParameters = Lists.newArrayList();
+    private List<String> positionalParameters = Lists.newArrayList();
 
     @Parameter(names = { "--cluster", "-c" },
             description = "Aliased Cluster Name")
-    String cluster;
+    private String cluster;
 
     @Parameter(names = { "--zkConnect", "-z" },
             description = "Zookeeper connection string for non aliased cluster")
-    String zkConnect;
+    private String zkConnect;
 
     @Parameter(names = { "--ls", "-l" },
             description = "Print child nodes")
-    boolean listChildren;
+    private boolean listChildren;
 
-    @Parameter(names = { "--rm-recursive", "-r" },
+    @Parameter(names = { "--rm-recursive", "-r", "--rmr" },
             description = "Remove a node, deleting any children recursively if necessary. Requires a path and cluster")
-    boolean deleteNodeRecursive;
+    private boolean deleteNodeRecursive;
 
     @Parameter(names = { "--rm" },
             description = "Remove a node only if it does not have children. Requires a path and cluster")
-    boolean deleteNodeNonRecursive;
+    private boolean deleteNodeNonRecursive;
 
-    @Parameter(names = { "--rm-children" },
+    @Parameter(names = { "--rm-children", "--rmc" },
             description = "Remove all children of a node recursively, keeping the specified node intact."
                     + "Requires a path and cluster")
-    boolean deleteChildrenOfNode;
+    private boolean deleteChildrenOfNode;
 
     @Parameter(names = { "--printPaths", "-p" },
             description = "When printing node names include full paths")
-    boolean printPaths;
+    private boolean printPaths;
 
     @Parameter(names = { "--get", "-g" },
             description = "Print data of path* requires a path and cluster")
-    boolean getData;
+    private boolean getData;
 
     @Parameter(names = { "--getMeta", "-m" },
             description = "Whether to print metadata* requires a path and cluster")
-    boolean getMeta;
+    private boolean getMeta;
 
     @Parameter(names = { "--metaField", "-f" },
             description = "A specific field of the stat object to get. Expects getter name")
-    String specificMetaFieldGetter;
+    private String specificMetaFieldGetter;
+
+    public List<String> getPositionalParameters() {
```

Comment:
I have but I'm against adding it here. I've found that if you're not an eclipse user your mileage varies wildly with some people wasting hours trying to make the IDE understand that those methods will appear later. There are some things for which Lombok might be worth adding but for getters and setters; they're easy enough to ignore and most IDE's generate them for you it is noise but it's not worth the hassle. Also I hear that particular feature may just be coming in Java 9.